commands/create.go

```diff
@@ -0,0 +1,111 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/tarantool/cartridge-cli/common"
+	"github.com/tarantool/cartridge-cli/create"
+	"github.com/tarantool/cartridge-cli/create/templates"
+	"github.com/tarantool/cartridge-cli/project"
+)
+
+const (
+	defaultTemplate = "cartridge"
+)
+
+func init() {
+	rootCmd.AddCommand(createCmd)
+
+	createCmd.Flags().StringVar(&projectCtx.Name, "name", "", "Application name")
+	createCmd.Flags().StringVar(
+		&projectCtx.Template,
+		"template",
+		templates.CartridgeTemplateName,
+		"Application template",
+	)
+}
+
+var createCmd = &cobra.Command{
+	Use:   "create [PATH]",
+	Short: "Create an application from the Cartridge template",
+	Long:  "Create an application in the specified PATH (default \".\")",
+	Args:  cobra.MaximumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		err := runCreateCommand(cmd, args)
+		if err != nil {
+			log.Fatalf(err.Error())
+		}
+	},
+}
+
+func runCreateCommand(cmd *cobra.Command, args []string) error {
+	setLogLevel()
+
+	var err error
+
+	// fill create-specific context
+	if err := fillCreateCtx(&projectCtx); err != nil {
+		return err
+	}
+
+	// get project path
+	basePath := cmd.Flags().Arg(0)
+	projectCtx.Path, err = getProjectPath(basePath)
+	if err != nil {
+		return err
+	}
+
+	// fill project-specific context
+	if err := project.FillCtx(&projectCtx); err != nil {
+		return err
+	}
+
+	// create project
+	if err := create.CreateProject(&projectCtx); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func fillCreateCtx(projectCtx *project.ProjectCtx) error {
+	// prompt name if not specified
+	if projectCtx.Name == "" {
+		projectCtx.Name = common.Prompt("Enter project name", "myapp")
+	}
+
+	return nil
+}
+
+func getProjectPath(basePath string) (string, error) {
```

Comment:
`nil, error` or `"", error`?

Reply:
Decided to use `"", error` in the whole project.